lib/Data.php

```diff
@@ -343,33 +343,33 @@ public function expire($expireDays = 365) {
 	 *    'field' => array('value', 'operator') => `field` operator 'value'
 	 */
 	public function deleteActivities($conditions) {
-		$sqlWhere = '';
-		$sqlParameters = $sqlWhereList = [];
+		$delete = $this->connection->getQueryBuilder();
```

Comment:
```
 1) OCA\Activity\Tests\DataDeleteActivitiesTest::testChunkingDeleteNotUsedWhenNotOnMysql
Error: Call to a member function delete() on null

/home/runner/work/activity/activity/apps/activity/lib/Data.php:347
```

Reply:
There is no way to fix the test, so I removed it.
It was mocking the DB connection and expecting the calls to prepare, rowCount, etc.
But since we execute the tests nowadays on all DBs, we can will also see when the delete does not work anymore